vuepress/docs/next/tutorials/consume/solr.md

```diff
@@ -0,0 +1,169 @@
+```
+helm install solr-operator apache-solr/solr-operator --version 0.5.0
+```
+4. Create a descriptor file `deployment-solrCloud.yaml` following the sample below. 
+
+``` yaml
+apiVersion: solr.apache.org/v1beta1
+kind: SolrCloud
+metadata:
+  name: solr
+spec:
+  solrAddressability:
+    external:
+      domainName: YOUR-HOSTNAME
+      method: Ingress
+      useExternalAddress: true
+  customSolrKubeOptions:
+    podOptions:
+      resources:
+        limits:
+          memory: 2Gi
+          cpu: 1000m
+        requests:
+          cpu: 350m
+          memory: 2Gi
+  dataStorage:
+    persistent:
+      pvcTemplate:
+        spec:
+          resources:
+            requests:
+              storage: 10Gi
+      reclaimPolicy: Delete
+  replicas: 3
+  solrImage:
+    repository: entando/entando-solr
+    tag: "8"
+  solrJavaMem: -Xms2048M -Xmx2048M
+  updateStrategy:
+    method: StatefulSet
+  zookeeperRef:
+    provided:
+      chroot: /explore
+      image:
+        pullPolicy: IfNotPresent
+        repository: pravega/zookeeper
+        tag: 0.2.13
+      persistence:
+        reclaimPolicy: Delete
+        spec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 1Gi
+      replicas: 3
+      zookeeperPodPolicy:
+        resources:
+          limits:
+            memory: 500Mi
+          requests:
+            cpu: 250m
+            memory: 500Mi
+
+```
+
+3. Apply the Solr application YAML, replacing `entando` with your namespace as needed:
+```
+kubectl apply -f deployment-solrCloud.yaml -n entando
+```
+4. To check that the service has started properly:
+```
+kubectl get solrclouds -w
+```
+You should see a response similar to this:
+```
+NAME   VERSION   TARGETVERSION   DESIREDNODES   NODES   READYNODES   UPTODATENODES   AGE
+solr   8                         1              1       1            1               79m
+```
+
+## Solr Standalone Integration
+
+### Create the Core
+1. Shell into the Solr pod:
+``` 
+kubectl exec -it YOUR-SOLR-POD-NAME -- /bin/bash
+```
+2. Generate the core:
+```
+bin/solr create_core -c YOUR-CORE-NAME
+```
+
+### Edit the Entando App Engine Deployment
+1. Scale down the App Engine deployment (entando-de-app image) to 0.
+2. Edit the App Engine deployment and add the following environment variables:
+
+```
+spec
+  -env
+    - name: SOLR_ACTIVE
+      value: "true"
+    - name: SOLR_ADDRESS
+      value: http://solr-solrcloud-0:80/solr
+```
+3. Scale the `entando-de-app` deployment back up to 1 and check the system.
+
+## Solr Integration for Multitenancy 
+
+### Generate the Core
+
+1. Generate the Solr core with the following command: 
+
+```
+curl http://YOUR-SOLR-INGRESS/solr/admin/collections?action=CREATE&name=YOUR-PRIMARY-CORE-NAME&numShards=1&replicationFactor=3&maxShardsPerNode=2
+```
+`YOUR-SOLR-INGRESS`= YOUR-NAMESPACE-solr-solrcloud.YOUR-HOSTNAME
+
+E.g., if your namespace is `entando` and your hostname is `k8s-entando.org`   
```

Comment:
Not sure this is true. For me, it was `YOUR-NAMESPACE-solr-solrcloud.YOUR-DOMAIN` where `YOUR-DOMAIN` is coming from the descriptor file edited below.

For that current example, it should be `entando-solr-solrcloud.entando.k8s-entando.org`

Another solution is to run the command `kubectl get ing` and get the first host for the name `solr-solrcloud-common`. I would rather use that solution because it is safer, and we already used that in the previous tutorials. Note that for Solr we have multiple hosts separated by commas, only the first one should be taken